core/tests/puppeteer-acceptance-tests/utilities/user/logged-in-user.ts

```diff
@@ -527,11 +791,16 @@ export class LoggedInUser extends BaseUser {
         img => img.src,
         profilePicture
       );
-      if (actualImageUrl !== expectedImageUrl) {
+
+      if (
+        actualImageUrl ===
+        '/assets/images/avatar/user_blue_150px.png?2983.800000011921'
```

Comment:
I think it's better to save the URL as a constant.

Reply:
Done.